crypto-ffi/src/wasm.rs

```diff
@@ -847,6 +847,15 @@ pub struct WireIdentity {
     /// MLS thumbprint
     #[wasm_bindgen(readonly, js_name = "thumbprint")]
     pub thumbprint: String,
+    /// X509 certificate serial number
+    #[wasm_bindgen(readonly, js_name = "serialNumber")]
```

Comment:
We need to fix this struct it seems. The `js_name` stuff doesn't work for structs and leads to `undefined`